src/Azure/composer.json

```diff
@@ -0,0 +1,54 @@
+{
+    "name": "open-telemetry/contrib-azure",
+    "description": "The Azure package for opentelemetry-php",
+    "keywords": ["opentelemetry", "otel", "azure", "tracing", "detector", "propagator"],
+    "type": "library",
+    "license": "Apache-2.0",
+    "authors": [
+        {
+            "name": "opentelemetry-php-contrib contributors",
+            "homepage": "https://github.com/open-telemetry/opentelemetry-php-contrib/graphs/contributors"
+        }
+    ],
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "require": {
+        "php": "^7.4 || ^8.0",
+        "guzzlehttp/guzzle": "^7.4",
+        "guzzlehttp/psr7": "^2.6",
+        "open-telemetry/api": "^1.0",
+        "open-telemetry/sdk": "^1.0",
+        "php-http/guzzle7-adapter": "^1.0"
```

Comment:
rather than requiring guzzle, it could use psr-18 discovery to find an http client (which is what the SDK does)